src/php/README.md

```diff
@@ -174,6 +174,16 @@ v1.4.0 | 3.3.0
 v1.6.0 | 3.4.0
 v1.8.0 | 3.5.0
 v1.12.0 | 3.5.2
+v1.13.1 | 3.5.2
+v1.14.2 | 3.5.2
+v1.15.1 | 3.6.1
+v1.16.1 | 3.6.1
+v1.17.2 | 3.6.1
+v1.18.0 | 3.6.1
+v1.19.1 | 3.6.1
+v1.20.1 | 3.7.0
```

Comment:
Thanks for updating the document!

Please sign the CLA.

The first release of each major version looks better IMO. For example, 1.20.0 instead of 1.20.1.
And v1.21.3 looks redundant here.

Reply:
I have signed CLA and it may take a while to take effect. I chose the last minor version in a major version because 1.20.1 is preferred to 1.20.0 for version 1.20.x.

Besides, do you think it would be better to move this table to the main README?

Reply:
I see. You are updating this table for gRPC in general instead of gRPC-PHP.

It would be useful if we can have such a table in the gRPC general document. @yashykt , WDYT and where is the appropriate place for putting such table?